libs/ngx-charts-on-fhir/src/lib/range-selector/range-selector.component.spec.ts

```diff
@@ -57,30 +57,34 @@ describe('RangeSelectorComponent', () => {
   });
 
   it('should calculate proper 1 month ago date from max layer date', async () => {
+    jasmine.clock().mockDate(new Date('2023-04-13'));
     let ButtonInputGroup = await loader.getHarness(MatButtonToggleHarness.with({ selector: "[id='1 mo']" }));
     await ButtonInputGroup.check();
-    const expectedMinDate = new Date('2022-02-28T00:00');
+    const expectedMinDate = new Date('2023-03-13');
```

Comment:
1. I chose those dates specifically to test an edge case in the month calculation with February having fewer days. Please don't change the dates. mockDate should use `'2022-03-30T00:00'` and expected date should be the same as before.
2. After changing that, the mock `timelineRange$.max` will be the same as the mocked current date. We should also change the mock timelineRange$ so the tests will fail if the buttons are using `timelineRange$.max`
3. I included the `T00:00` because I wanted these dates to be treated as local time. Date constructor treats date-only string as UTC, which made the tests fail in some timezones. Why did this need to be changed to date-only? Does mockDate affect the timezone?